products/bash/release-mq.sh

```diff
@@ -151,9 +148,25 @@ EOF
 
   echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
 
-  # -------------------------------------- CHECK FOR NEW IMAGE DEPLOYMENT STATUS ------------------------------------------
+  # --------------------------------------------------- FIND IMAGE TAG ---------------------------------------------------
+
+  echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+
+  echo "INFO: Image is '$image_name'"
+  echo "INFO: Release name is: '$release_name'"
 
-  echo "INFO: Image tag for '$release_name' is '$imageTag'"
+  imageTag=${image_name##*:}
+  
+  echo "INFO: Image tag found for '$release_name' is '$imageTag'"
+
+  if [[ -z "$imageTag" ]]; then
```

Comment:
As above, best to check this near the top, validate parameters as soon as possible.

Reply:
Done